examples/demoPool.py

```diff
@@ -8,12 +8,11 @@
 #
 
 from __future__ import print_function
-from builtins import map
+import math
 from builtins import range
```

Comment:
You can remove the `builtins` imports and the `__future__` while you are here. We don't need py2 compatibility.

Reply:
Removed `builtins` and `__future__` from all files in this repo.